src/Pelagos/Bundle/AppBundle/Command/DoiDatacitePostMigrationCommand.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Pelagos\Bundle\AppBundle\Command;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\ORMException;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use Pelagos\Entity\Dataset;
+use Pelagos\Entity\DatasetSubmission;
+use Pelagos\Entity\DOI;
+use Pelagos\Entity\DIF;
+
+use Pelagos\Util\DOIutil;
+
+/**
+ * Post-migration Script for moving from EZid Library to Datacite.
+ *
+ * @see ContainerAwareCommand
+ */
+class DoiDatacitePostMigrationCommand extends ContainerAwareCommand
+{
+    /**
+     * Configures the current command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('dataset-doi:post-migration-script')
+            ->setDescription('Recreate DOIs for DIF approved datasets.');
+    }
+
+    /**
+     * Executes the current command.
+     *
+     * @param InputInterface  $input  An InputInterface instance.
+     * @param OutputInterface $output An OutputInterface instance.
+     *
+     * @throws \Exception When dataset not found.
```

Comment:
These exceptions are not in this functions.

Reply:
Removed it